### PR TITLE
[Ready] Policy config refactor for Forgottenship ruin.

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

Policy config isn't replacing "important_info" in ghost-role spawner objects now, but instead, it will be showing up as the only, smaller text you see once you spawned in.
It also sets "show_flavour" to 'false' so you don't have to read 2 text walls, focusing your attention on policy.

## Why It's Good For The Game

You can put large walls of text in config, so people will know every single detail of it.

Screenshot_85

## Changelog
:cl: EgorDinamit
tweak: Policy config for cybersun ship ruin is now the only text you will see when you spawn as its crew.
/:cl:

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
what's the point of the flavor text if you cannot see it? 

A Wall of Text is really NBD in a role where you can't use anything above that wall of text to begin with ;)

**Comment by EgorDinamit** [comment]
@ExcessiveUseOfCobblestone you can see flavour text in spawners menu, once you are spawned - you see policy about what you really should or shouldn't do.
If you are here for lore - you open ghost role spawners and read, otherwise you just teleport and click.

**Comment by ExcessiveUseOfCobblestone** [comment]
Fair.

That said, is this role actually policy-supported? In particular, if I spawn in as this role can I click on the verb and it print out what it prints out at the beginning?

If not that will need to be changed (not necessarily in this PR). I am beginning to worry that this approach to just get policy keyword followed by a to_chat is the wrong solution the ideal approach is all mobs on login/start/etc. print all policy associated with their character and continues to do so when utilizing the verb (the verb support being more important than the start message)

**Comment by EgorDinamit** [comment]
Tbh AI and Cyborgs have the same problem if they didn't read through whole silicon policy AND headmins' rulings.
Policy support was added in previous PR, but headmins made particularly long text for important_info.
Your idea sounds like what we have with payer's memory(Notes: Objectives, Account ID).

**Comment by ExcessiveUseOfCobblestone** [comment]
No my point was I think it's being done incorrectly if the only "support" is an on-spawn to_chat.

You should be able to access relevant policy at any time through the already existing verb.

**Comment by EgorDinamit** [comment]
What is the said verb?

**Comment by ExcessiveUseOfCobblestone** [comment]
https://github.com/tgstation/tgstation/blob/06e440ad32f83f78f84b1ec2c1b6b80af21894ac/code/modules/client/verbs/ooc.dm#L364

**Comment by EgorDinamit** [comment]
Alright, give me some time, I'll change it to this verb.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -647,43 +647,37 @@
...
 	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy hands."
 	outfit = /datum/outfit/syndicatespace/syndicrew
-	assignedrole = "Cybersun Crewmember"
-
-/obj/effect/mob_spawn/human/syndicatespace/Initialize(mapload)
-	. = ..()
-	var/policy = get_policy(ROLE_SYNDICATE_CYBERSUN)
-	if(policy)
-		important_info = policy
+	assignedrole = "Cybersun Space Syndicate"
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/structures/ghost_role_spawners.dm:657]: you should set this string as the define for `ROLE_SYNDICATE_CYBERSUN` (and similarly for the captain), then you can have in special `get_policy(assignedrole)` so you aren't copypasting for the captain

Comment (EgorDinamit) [reply code/game/objects/structures/ghost_role_spawners.dm:657]: "Define" will change it in game to "Cybersun Space Syndicate" anyway, right?

Comment (EgorDinamit) [reply code/game/objects/structures/ghost_role_spawners.dm]: It's done.

**Pull request merged**